index.js

```diff
@@ -127,9 +127,8 @@ class SauceBrowser extends Browser {
     if (Object.keys(caps).length === 0) {
       return process.nextTick(cb, new Error('Capabilities are required'))
     }
-
     const testName = options.name || this.manifest.name
-    const build = buildNumber()
+    const build = buildNumber() || `${process.env.GITHUB_RUN_ID}-${process.env.GITHUB_RUN_NUMBER}`
```

Comment:
```suggestion
    const build = buildNumber() || process.env.GITHUB_RUN_ID
```

Unless it causes issues in Sauce Labs, I prefer just `process.env.GITHUB_RUN_ID` so that:

1. It can later be moved to [`build-number`](https://github.com/nkt/build-number), where e.g. `TRAVIS_BUILD_NUMBER` seems to have the same meaning as our `GITHUB_RUN_ID`
2. Doesn't result in `-` for CI providers (or local use) that are neither GHA or supported by `build-number`.

Reply:
Sure, I'll make the change in a few and re-push.

Reply:
Added the update